### PR TITLE
Add Github Actions and PR template

### PR DESCRIPTION
**Pull request by jarosenb**
Adds Github housekeeping.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Overview:
+
+## Related Github Issues:
```
Comment (wesleyboar) [inline .github/pull_request_template.md:3]: Are we using "GitHub" issues?

How about just:
```suggestion
## Related:
```

This way we can add related JIRA tickets, GItHub issues, or PRs. Example (https://github.com/TACC/Core-CMS/pull/495):

tacc-core-cms-495

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- [TUP-1234](https://jira.tacc.utexas.edu/browse/TUP-1234)
+
+## Summary of Changes:
+
+## Testing Steps:
```
Comment (wesleyboar) [inline .github/pull_request_template.md:9]:
```suggestion
## Testing:
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+1.
+
+## UI Photos:
```
Comment (wesleyboar) [inline .github/pull_request_template.md:13]:
```suggestion
## UI:
```

**Review by wesleyboar — COMMENTED** [review]
I will suggest these simplification of PR templates for Portal, too. I made these simplifications in CMS. I find fewer words allow easier skimming.

**Review by jchuahtacc — APPROVED** [review]
LGTM

**Pull request merged**